.gitignore

```diff
@@ -71,7 +71,6 @@ dlldata.c
 *.tlh
 *.tmp
 *.tmp_proj
```

Comment:
Is this an intentional change to remove  *.log?

Reply:
For PublishTestResultsV1/Tests/data/a.log and PublishTestResultsV1/Tests/data/rec/a.log, the validation failed since we git ignore *.log files. Git only identified ones under Tasks/PublishTestResultsV1/Tests/data/a.log, but not _generated/PublishTestResultsV1/Tests/data/a.log etc.

Reply:
Can we narrowly allow that file e.g. *.log under *PublishTestResultsV1/Tests/Data/Rec*, rather than everywhere?  Better to avoid ending up checking in a bunch of *.log files.